include/vara/Feature/Feature.h

```diff
@@ -159,8 +159,8 @@ class Feature {
   [[nodiscard]] FeatureSourceRange *getFeatureSourceRange() {
     return Source.has_value() ? &Source.value() : nullptr;
   }
-  void setFeatureSourceRange(FeatureSourceRange FeatureSR) {
-    Source = std::move(FeatureSR);
+  void setFeatureSourceRange(FeatureSourceRange &FeatureSR) {
+    Source = FeatureSourceRange(FeatureSR);
```

Comment:
```suggestion
  void setFeatureSourceRange(FeatureSourceRange FeatureSR) {
    Source = std::move(FeatureSR);
```
The original should actually be correct. The FeatureSR is copied into this function, therefore, we are allowed to move it. If I revert you change here,. all feature-unitttests and python tests pass on my system.

Could you please revert it locally and double check if the test fails afterwards.

Reply:
The original seems to work. I am not sure why it didn't work before. Maybe I forgot to update the pip package...

Reply:
Maybe, but at least we now have a test for it :)